lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
```

Comment:
Shouldn't it be better to add `.freeze` to other constants? It would prevent some possible weird cases. Ex. when some other code will use `gsub!` instead of `gsub`, and same of a kind.